src/containers/Profile.jsx

```diff
@@ -42,17 +42,22 @@ const mapDispatchToProps = (dispatch, ownProps) => ({
     dispatch(desactivate2FA(mode))
   },
   onPassphraseSimpleSubmit: (current, newVal) => {
-    return dispatch(updatePassphrase(current, newVal)).catch(
```

Comment:
Ok now I see why it did not work. Replacing the `catch` by a `then` should make it.

Reply:
No, we have to call the alerter before the setTimeout so before the promise return. It's why I used a callback here because `then` was not working correctly.

Reply:
I need to check by myself because to me the `setTimeout` is asynchronous, and we already fulfilled the promise when it is called.